File: fixtures/dart_output/get_with_single_header.dart
import 'package:http/http.dart' as http;

void main() async {
  var headers = {
    'foo': 'bar',
  };

  var res = await http.get('http://example.com/', headers: headers);
  if (res.statusCode != 200) throw Exception('get error: statusCode= ${res.statusCode}');
  print(res.body);
}

File: fixtures/scrapy_output/get_with_single_header.py
from scrapy import Request, Spider


class MySpider(Spider):
    name = 'myspider'

    def start_requests(self):
        headers = {
            'foo': 'bar',
        }
        yield Request(
            'http://example.com/',
            callback=self.parse,
            headers=headers,
        )

    def parse(self, response):
        pass
